modules/installing-python-packages-on-your-code-server-workbench.adoc

```diff
@@ -0,0 +1,70 @@
+:_module-type: PROCEDURE
+
+[id="installing-python-packages-on-your-code-server-workbench_{context}"]
+= Installing Python packages on your code-server workbench
+
+[role='_abstract']
+You can install Python packages that are not part of the default workbench image by adding the package and the version to a `requirements.txt` file and then running the `pip install` command in a terminal window.
+
+ifndef::upstream[]
+NOTE: You can also install packages directly, but {org-name} recommends using a `requirements.txt` file so that the packages stated in the file can be easily re-used across different notebooks. In addition, using a `requirements.txt` file is also useful when using a S2I build to deploy a model.
+endif::[]
+ifdef::upstream[]
+NOTE: You can also install packages directly, but using a `requirements.txt` file so that the packages stated in the file can be easily re-used across different notebooks is recommended. In addition, using a `requirements.txt` file is also useful when using a S2I build to deploy a model.
+endif::[]
+
+.Prerequisites
+* A launched and running code-server workbench.
+
+.Procedure
+. In your code-server workbench, from the Activity Bar, select the menu icon (image:images/codeserver-menu-icon.png[The Menu icon]) > *File* > *New Text File* to create a new text file.
+. Add the packages to install to the text file.
++
+[source]
+----
+altair
+----
++
+You can specify the exact version to install by using the `==` (equal to) operator, for example:
++
+[source]
+----
+altair==4.1.0
+----
++
+ifndef::upstream[]
+[NOTE]
+====
+{org-name} recommends specifying exact package versions to enhance the stability of your workbench over time. New package versions can introduce undesirable or unexpected changes in your environment's behavior.
+====
+endif::[]
+ifdef::upstream[]
+Specifying exact package versions to enhance the stability of your workbench over time is recommended. New package versions can introduce undesirable or unexpected changes in your environment's behavior. 
```

Comment:
Consider rewording and then you might not need to conditionalize the note:

"Best practice it so specify a package version to enhance..."